lighthouse-core/lib/i18n/locales/en-US.json

```diff
@@ -276,10 +276,10 @@
     "message": "Screen readers have features to make navigating tables easier. Ensuring `<td>` cells using the `[headers]` attribute only refer to other cells in the same table may improve the experience for screen reader users. [Learn more](https://web.dev/td-headers-attr/)."
   },
   "lighthouse-core/audits/accessibility/td-headers-attr.js | failureTitle": {
-    "message": "Cells in a `<table>` element that use the `[headers]` attribute do not refer to headers of the same table."
+    "message": "Cells in a `<table>` element that use the `[headers]` attribute refer to element `id`s not found within the same table."
```

Comment:
@exterkamp is the plurliazation of a placeholder not a problem then? I noticed you did it in #9590 too

Reply:
Shouldn't be, it's only a problem when we have a variable amount of things.  If we passed in a variable like `idCount` and needed a variable amount of `id`s then it would be a problem.

Reply:
> Shouldn't be, it's only a problem when we have a variable amount of things. If we passed in a variable like `idCount` and needed a variable amount of `id`s then it would be a problem.

but is there a single character (or equivalent characters) in other languages that means "make the attached word plural"?

Seems better to reword, even if it gets into plural awkwardness (and even better if we can reword to avoid the awkwardness) `"refer to an element `id` not found within the same table"` or whatever

Reply:
Gotcha.  Done!

Reply:
ya brendan much better worded my concern :) 👍